feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -155,6 +155,9 @@ $sapFioriContent_LabelColor: #6a6d70;
       padding-bottom: 0;
     }
     max-height: calc(100vh - 260px);
+    @include media-breakpoint-down(xs) {
```

Comment:
@include `media-breakpoint-down(xs)`
means the following class only works when the screen size is below the breakpoint of `xs`